Backend/scripts/build-size.sh

```
@@ -0,0 +1,33 @@
+  exit 1
+fi
+
+echo "=== Total size ==="
+du -sh "$BUILD"
+echo
+
+echo "=== Size by extension (top 20) ==="
+find "$BUILD" -type f -name '*.*' 2>/dev/null | \
+  sed 's/.*\.\([^./]*\)$/\1/' | \
+  sort | uniq -c | sort -rn | head -20 | \
+  while read count ext; do
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)